### PR TITLE
Updating manifest to have all the kubernetes features

### PR DESCRIPTION
**Pull request by CharlyF**
### What does this PR do?

Updating manifest to have all the kubernetes features

### Motivation

consistent with the docs

### Additional Notes

@l0k0ms

### CODE REVIEW

File: Dockerfiles/manifests/agent.yaml [hunk 1]
```diff
@@ -8,17 +8,58 @@ spec:
...
+          - name: procdir
+            mountPath: /host/proc
+            readOnly: true
+          - name: cgroups
+            mountPath: /host/sys/fs/cgroup
+            readOnly: true
+        livenessProbe:
+          exec:
+            command:
+            - ./probe.sh
```
Comment (JulienBalestra) [inline Dockerfiles/manifests/agent.yaml]: Why not using the healthcheck ?

Comment (mfpierre) [reply Dockerfiles/manifests/agent.yaml]: @JulienBalestra it is https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/agent/probe.sh

Comment (CharlyF) [reply Dockerfiles/manifests/agent.yaml]: 👍

File: Dockerfiles/manifests/agent.yaml [hunk 2]
```diff
@@ -8,17 +8,58 @@ spec:
...
       name: datadog-agent
-      namespace: default
     spec:
       serviceAccountName: datadog-agent
       containers:
       - image: datadog/agent:latest
         imagePullPolicy: Always
         name: datadog-agent
+        ports:
+          - containerPort: 8125
```
Comment (mfpierre) [inline Dockerfiles/manifests/agent.yaml]: You should add also
```yaml
- containerPort: 8126
            name: traceport
            protocol: TCP
```

**Review by mfpierre — CHANGES_REQUESTED** [review]

**Review by mfpierre — APPROVED** [review]
☸️

**Pull request merged**